server/server_test.go

```diff
@@ -85,3 +89,54 @@ func TestGetKeysEndpoint(t *testing.T) {
 		assert.Equal(t, expectedStatus, res.StatusCode)
 	}
 }
+
+// This just checks the URL routing is working correctly.
+// More detailed tests for this path including negative
+// tests are located in /server/handlers/
+func TestGetRoleByHash(t *testing.T) {
+	store := storage.NewMemStorage()
+
+	ts := data.SignedTimestamp{
+		Signatures: make([]data.Signature, 0),
+		Signed: data.Timestamp{
+			Type:    data.TUFTypes["timestamp"],
+			Version: 0,
+			Expires: data.DefaultExpires("timestamp"),
+		},
+	}
+	j, err := json.Marshal(&ts)
+	assert.NoError(t, err)
+	update := storage.MetaUpdate{
+		Role:    data.CanonicalTimestampRole,
+		Version: 1,
+		Data:    j,
+	}
+	checksumBytes := sha256.Sum256(j)
+	checksum := hex.EncodeToString(checksumBytes[:])
+
+	store.UpdateCurrent("gun", update)
```

Comment:
Would it be possible to add a second timestamp in here?  Since there is only one timestamp in the store right now, and it's the current one, this test would still pass if checksum returned only the current timestamp.